### PR TITLE
Add tox tests

### PR DESCRIPTION
**Pull request by tristan-cognite**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -47,10 +47,12 @@ podTemplate(
                 sh("pip3 install poetry")
             }
             stage('Install all dependencies') {
+                sh("pyenv install 3.6.6 && pyenv install 3.7.2")
```
Comment (eventh) [inline Jenkinsfile]: I thought the we are using the multi-python docker image for this that comes with the right python versions already installed. If you already checked that, ignore this comment :)

**Comment by eventh** [comment]
Covers #49

File: Jenkinsfile (hunk 1 above)
Comment (eventh) [reply Jenkinsfile]: https://github.com/cognitedata/multi-python/blob/master/Dockerfile
    pyenv install 3.5.0 && \
    pyenv install 3.6.6 && \
    pyenv install 3.7.2

Comment (tristan-cognite) [reply Jenkinsfile]: Nice find, I did not notice that.

**Comment by tristan-cognite** [comment]
trying to replicate error locally

**Pull request merged**